simple_history/models.py

```diff
@@ -579,6 +579,8 @@ def __init__(self, model, fields_included):
         self.fields_included = fields_included
 
     def __get__(self, instance, owner):
+        if instance is None:
```

Comment:
@hwalinga Why would you want to return the descriptor?

Reply:
Oh I see your stackoverflow post. Got it.

Reply:
Yes, it is so that any metaprogramming done on the class knows what kind of attribute this is. (ie a descriptor class) You can also return None, but this way you allow anyone inspecting the class (programmatically) to know what is in here. 

That is also the bug I got in DRF. There was some metaprogramming that failed to continue when it got to this attribute.